comps.pm

```diff
@@ -131,7 +133,11 @@ sub do_comps {
   $h{AOACINTT}[1] = ${$h{AOACINTT}}[1]/1000;
 
   # if shld hv is off, make rate 0, acorn comes out NaN
-  if (${$h{"2S2HVST"}}[1] == 0) { ${$h{"2SHLDART"}}[1]=0; }
+  if (${$h{"2S2HVST"}}[1] == 0) { ${$h{"2SHLDBRT"}}[1]=0; }
+
+  # Convert to high order byte rates
```

Comment:
Sorry for being blunt, but this is a classic example of a bad comment. It's obvious from the next line that the high-order byte is being extracted. What would be useful is to explain why, possibly including a link to a notebook if that helps. The next line deserves a separate comment as well.